.claude/skills/github/scripts/issue/Set-IssueAssignee.ps1

```diff
@@ -0,0 +1,95 @@
+<#
+.SYNOPSIS
+    Assigns users to a GitHub Issue.
+
+.DESCRIPTION
+    Manages assignees on GitHub Issues:
+    - Adds one or more assignees to an issue
+    - Supports @me shorthand for current authenticated user
+    - Validates assignees are valid GitHub usernames
+
+.PARAMETER Owner
+    Repository owner. Inferred from git remote if not provided.
+
+.PARAMETER Repo
+    Repository name. Inferred from git remote if not provided.
+
+.PARAMETER Issue
+    Issue number (required).
+
+.PARAMETER Assignees
+    Array of GitHub usernames to assign. Use "@me" for current user.
+
+.EXAMPLE
+    .\Set-IssueAssignee.ps1 -Issue 123 -Assignees @("@me")
+    .\Set-IssueAssignee.ps1 -Issue 123 -Assignees @("user1", "user2")
+
+.NOTES
+    Exit Codes: 0=Success, 1=Invalid params, 3=API error, 4=Not authenticated
+#>
+
+[CmdletBinding()]
+param(
+    [string]$Owner,
+    [string]$Repo,
+    [Parameter(Mandatory)] [int]$Issue,
+    [Parameter(Mandatory)] [string[]]$Assignees
+)
+
+Import-Module (Join-Path $PSScriptRoot ".." ".." "modules" "GitHubHelpers.psm1") -Force
+
+Assert-GhAuthenticated
+$resolved = Resolve-RepoParams -Owner $Owner -Repo $Repo
+$Owner = $resolved.Owner
+$Repo = $resolved.Repo
+
+# Resolve @me to actual username

```

Comment:
@rjmurillo-bot why is the resolution of `@me` to an identity required? It's supported by the API to use the current identity. This burns a few API requests and adds code and complexity for what strategic purpose?